android/assets/jsons/translations/template.properties

```diff
@@ -1256,9 +1256,10 @@ Unlock =
 Move to city = 
 Reset Citizens = 
 Citizen Management = 
+Citizen Focus = 
 Avoid Growth = 
-Default Focus = 
-[stat] Focus = 
+# Default = 
```

Comment:
Should this be a comment...?

Reply:
That was intentional since Default already gets translated elsewhere. I can elaborate more

Reply:
Then this should be removed, not commented